routes/web.php

```diff
@@ -22,3 +22,8 @@
 Route::get('/home', 'HomeController@index')->name('home');
 
 Route::get('/admin/ama-list/', 'Admin\AmaController@index')->middleware('auth');
+
+Route::get('/create', 'Admin\AmaController@create')->middleware('auth');
```

Comment:
Use Laravel resource routes instead of creating each route separately. Check [this](https://laravel.com/docs/5.7/controllers#resource-controllers).